### ISSUE TITLE
video

### ISSUE THREAD

**Issue by BILL-CS** [Milkistasty/Practical-Machine-Learning#1]
Do you have any videos of this course?

**Comment by Milkistasty** [comment]
https://c.d2l.ai/stanford-cs329p/

**Comment by BILL-CS** [comment]
> https://c.d2l.ai/stanford-cs329p/

I konw this website, but there are only slides in it, the link of video is empty and unavailable. Am I right? Or is there something wrong with me?
I can find some videos on Youtube and bilibili, but not all of them. Do you have or could you find all the videos?

**Comment by Milkistasty** [comment]
yeah, I also used the videos on Bilibili for study. And I don't know if there are extra videos~
